### ISSUE TITLE
Multilingual comments problem

### ISSUE THREAD

**Issue by stoner-city** [exstreme/Jcomments-Recaptcha2#19]
On a multilingual site comments on the same article (language: all) are displayed only in the actual language. How to show all comments in different languages at once?

**Comment by exstreme** [comment]
Hello!
Did you edit settings for each language in Jcomments?

**Comment by stoner-city** [comment]
Yes I did. But on the german site only the german comments are shown and vice versa on the english site - have a look at the screenshots.
![german](https://user-images.githubusercontent.com/61195723/74749700-d8786000-526a-11ea-931f-e65cecd82702.png)
![english](https://user-images.githubusercontent.com/61195723/74749717-dc0be700-526a-11ea-8e9c-d8aafd321a3f.png)

**Comment by exstreme** [comment]
Recaptcha isn't work on hidden form. Can you take screenshots with jcomments captcha settings in English and Germany version ?

**Comment by stoner-city** [comment]
I think we are talking about two different things?! The captcha is working fine what I need is that on an article set to all languages shows all comments that where posted no matter what language. I have made a screenshot from the db have a look at object_is 38. The comments are posted on the same site but a German user can only see the German comments and the English user only the English comments. I want to show all comments on this article. If I uncomment the Multilingual query in models/jcomments.php (line 291) it will do what I want.
`if (JCommentsMultilingual::isEnabled()) {
			$language = isset($options['lang']) ? $options['lang'] : JCommentsMultilingual::getLanguage();
			$where[] = "c.lang = " . $db->Quote($language);
		}`
![db__jcomments](https://user-images.githubusercontent.com/61195723/74814867-79612c80-52f8-11ea-8d99-dd109faf4f48.png)
![en_site](https://user-images.githubusercontent.com/61195723/74814881-7e25e080-52f8-11ea-8233-e6eb4a41fcad.png)
![de_site](https://user-images.githubusercontent.com/61195723/74814887-80883a80-52f8-11ea-8f1b-32b6bee3ab93.png)

**Comment by exstreme** [comment]
I think, you need using separate plugin, which can allow all comments

**Comment by Globulopolis** [comment]
> I think, you need using separate plugin, which can allow all comments

No. JComments save a site language for comment, not an article language.
We can fix it in https://github.com/exstreme/Jcomments-Recaptcha2/blob/Develop/site/classes/multilingual.php#L57 by adding database query to select article language.
